adobe/manip.hpp

```diff
@@ -18,11 +18,17 @@
 
 #include <iostream>
 
+/*
+    REVISIT (sean-parent): As far as I can determine, this file does not use adobe/serializable.hpp
+    However, if it is not included here, the build fails for C++17 under MSVC only. I have not been
```

Comment:
If include order matters in this file, I think you could be clearer about "here."  Could be interpreted to mean “in this file.”